emcc.py

```diff
@@ -1283,6 +1279,28 @@ def run(args):
     print(shared.get_llvm_target())
     return 0
 
+  if '-print-search-dirs' in newargs:
```

Comment:
Noticed one tiny bug after installing this from tot: in Clang those arguments are accepted as both `-print-search-dirs` and `--print-search-dirs` (and same for others), but here it only works with `-print` form.